packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -385,9 +385,11 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			0x384 /* requesting handle for unknown blob */,
 		);
 		const pending = this.pendingBlobs.get(id);
+		// Create a callback function to handle the attachment of the blob
```

Comment:
This wording is clearer to me, if it's accurate :)
```suggestion
		// Create a callback function for once the blob has been attached
```